docs/experiments-clir.md

```diff
@@ -0,0 +1,52 @@
+# Cross-lingual Information Retrieval Experiments
```

Comment:
See https://github.com/castorini/anserini/blob/master/src/main/resources/docgen/templates/robust04.template for an example of how this is actually done.

Please make consistent?

Title should be collection-specific (e.g., NTCIR).